create-presigned-download-url/src/main/java/no/unit/nva/download/publication/file/publication/RestPublicationService.java

```diff
@@ -0,0 +1,99 @@
+package no.unit.nva.download.publication.file.publication;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.mikael.urlbuilder.UrlBuilder;
+import no.unit.nva.download.publication.file.publication.exception.NoResponseException;
+import no.unit.nva.download.publication.file.publication.exception.NotFoundException;
+import no.unit.nva.model.Publication;
+
+import nva.commons.exceptions.ApiGatewayException;
+import nva.commons.utils.Environment;
+import nva.commons.utils.JsonUtils;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.UUID;
+
+public class RestPublicationService {
+
+    public static final String PATH = "/resource/";
```

Comment:
You should request GET /publication/{identifier} instead, that will give you the Publication object and not the raw DynamoDB response.